packages/styles/scss/patterns/leadspace/_leadspace.scss

```diff
@@ -137,9 +138,13 @@
   }
 
   @include carbon--breakpoint(max) {
-    .#{$prefix}--leadspace__overlay {
-      padding-top: $spacing-08;
-      padding-bottom: $spacing-08;
+    .#{$prefix}--leadspace {
+      padding-top: carbon--mini-units(80) / carbon--mini-units(198) * 100%;
```

Comment:
Just out of curiosity, what aspect ratios are we using? If I did my math right this one reduced down to `40:99`. I don't see it on our list of aspect ratios?

https://www.carbondesignsystem.com/guidelines/layout#aspect-ratio

Reply:
hmmm right

@larahanlon2 are we following the aspect ratio guidelines?

Reply:
@photodow talked to @larahanlon2 and @wonilsuhibm and we decided if we followed the ratios in the guidelines it would get too large so we're sticking to the aspect ratios in the specs